src/colorprint.cc

```diff
@@ -89,7 +89,11 @@ std::string FormatString(const char* msg, va_list args) {
 
   std::size_t size = 256;
   char local_buff[256];
+#ifdef __ANDROID__
+  auto ret = ::vsnprintf(local_buff, size, msg, args_cp);
```

Comment:
is this part of a broader set of checks that should be guarded by  #if __cplusplus >= 201103L or is it truly Android specific?